ckanext/harvest/plugin.py

```diff
@@ -36,14 +37,37 @@ class Harvest(p.SingletonPlugin, DefaultDatasetForm):
 
     ## IPackageController
 
+    def _analyze_to_run_externally(data_dict):
+        # if use_external_harvester_app is True we set and inactive locally
```

Comment:
s/we set and inactive/we set as inactive/ ?

Reply:
[fixed](https://github.com/avdata99/ckanext-harvest/commit/735beb23b1739f541d91a486d2000da20b7afb0e)